checkov/terraform/checks/resource/azure/AzureFrontDoorEnablesWAF.py

```diff
@@ -12,7 +12,7 @@ def __init__(self):
         super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
 
     def get_inspected_key(self):
-        return "web_application_firewall_policy_link_id"
+        return "web_application_firewall_policy_link_id" or "frontend_endpoint/[0]/web_application_firewall_policy_link_id"
```

Comment:
```suggestion
        return "frontend_endpoint/[0]/web_application_firewall_policy_link_id"
```
the documentation doesn't even suggest, that it is possible to add a `web_application_firewall_policy_link_id` argument in the root level, so lets remove it and then of course remove the test, which expects the wrong way to be correct.

Reply:
Good catch! Updated.